src/app/layout/header/header.component.html

```diff
@@ -0,0 +1,41 @@
+<header class="py-4">
+  <div class="flex space-x-8 justify-between items-center">
+    <ul class="flex space-x-8">
+      <li *ngFor="let link of contactsLinks">
+        <a [href]="link.href" class="text-sm">{{ link.text }}</a>
+      </li>
+    </ul>
+    <ul class="flex space-x-8">
+      <li *ngFor="let link of infoLinks">
+        <a [href]="link.href" class="text-sm">{{ link.text }}</a>
+      </li>
+    </ul>
+  </div>
+  <div class="flex justify-between items-center my-6">
+    <img src="assets/logo.svg" alt="Logo image" />
+    <div class="flex space-x-4">
+      <app-auth></app-auth>
+      <app-cart></app-cart>
+    </div>
+  </div>
+  <div>
+    <ul class="flex justify-between items-center bg-gray-50 px-8">
+      <li
+        *ngFor="let category of categories"
+        class="relative py-6 hover:text-main-green"
+        (mouseenter)="setActiveCategory(category.name)"
```

Comment:
This is correct. 
As an alternative, if the method is such short, you can directly in HTML assign values: 
```suggestion
        (mouseenter)="activeCategory = category.name"
        (mouseleave)="activeCategory = null"
```
You don't have to do any changes here, just for general knowledge